config/core/test-configs.yaml

```diff
@@ -210,6 +210,18 @@ test_plans:
       job_timeout: '10'
       kselftest_collections: "lib"
 
+  kselftest-lkdtm:
+    <<: *kselftest
+    params:
+      job_timeout: '10'
+      kselftest_collections: "lkdtm"
+
+kselftest-seccomp:
```

Comment:
```suggestion
  kselftest-seccomp:
```

Reply:
Whoops! I'm not sure how that happened. Thanks for catching that.